test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
```

Comment:
__[development][rubocop]__
* Method definitions must not be nested. Use `lambda` instead. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-methods)

<!-- Comment by [SideCI](https://sideci.com) -->